proof/refine/ARM/Ipc_R.thy

```diff
@@ -243,7 +243,7 @@ lemma corres_set_extra_badge:
           (\<lambda>_. msg_max_length + 2 + n < unat max_ipc_words))
          (set_extra_badge buffer b n) (setExtraBadge buffer b' n)"
```

Comment:
It looks like this one has been missed. It’s currently called `corres_set_extra_badge`.

Reply:
Missed because only searched for `corres:`, but I'll include the change